Src/FluentAssertions/Specialized/ActionAssertions.cs

```diff
@@ -172,6 +144,45 @@ public void NotThrowAfter(TimeSpan waitTime, TimeSpan pollInterval, string becau
                 .FailWith("Did not expect any exceptions after {0}{reason}, but found {1}.", waitTime, exception);
         }
 
+        protected ExceptionAssertions<TException> Throw<TException>(Exception exception, string because, object[] becauseArgs)
+            where TException : Exception
+        {
+            IEnumerable<TException> expectedExceptions = extractor.OfType<TException>(exception).ToArray();
+
+            Execute.Assertion
+                .ForCondition(exception != null)
+                .BecauseOf(because, becauseArgs)
+                .FailWith("Expected a <{0}> to be thrown{reason}, but no exception was thrown.", typeof(TException));
+
+            Execute.Assertion
+                .ForCondition(expectedExceptions.Any())
+                .BecauseOf(because, becauseArgs)
+                .FailWith(
+                    "Expected a <{0}> to be thrown{reason}, but found <{1}>: {3}.",
```

Comment:
🙃 The format string does not use `Environment.NewLine` and `{2}`.

Reply:
Sorry this slipped through. I'm surprised the tests didn't pick this up. 
I'll look into this right now. Thanks!

Reply:
Sorry about this. I just looked into it and found that this was how things looked before, but in a different area of ActionAssertions.
I'll place it directly. before the {3}

Reply:
No worries, they've snuck in during refactorings. 
I'm aware of at least one more format string in the repo that has similar "typo".

Reply:
It does seem that way. This pull request is a good opportunity to resolve all of those at least :)

Reply:
@jnyrup The other typo isn't in the same area that I changed for this feature, so I've resolved this conversation and I'd recommend we fix that typo in another PR